src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -82,8 +82,18 @@ struct KeyValueViewModel {
         let type = constantSchema.valueType.type
 
         if type == .string {
-            self.value = "\"\(constantValue)\""
-            self.constantValue = "\"\(constantValue)\""
+            var skipUrlEncoding: Bool?
+            var value: String
+            if let value = param.value.extensions?["x-ms-skip-url-encoding"] {
+                skipUrlEncoding = value.value as? Bool
+            }
+            if skipUrlEncoding ?? false {
+                value = "\"\(constantValue)\".removingPercentEncoding ?? \"\(constantValue)\""
```

Comment:
I would expect that if `x-ms-skip-url-encoding` is true, then using the right-hand side as a fallback would fail. Perhaps this should just fail, or empty string?

Reply:
Yeah. Change to empty string.